powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthEnrollmentConfigurationTest.java

```diff
@@ -94,10 +93,10 @@ void testConfiguration() throws Exception {
                 .findFirst()
                 .orElseThrow(() -> new AssertionError("error - no configuration found"));
 
-        assertThat(response, hasProperty("mobileApplication.android.minimalVersion", equalTo("1.4.0")));
-        assertThat(response, hasProperty("mobileApplication.android.currentVersion", equalTo("1.5.4")));
-        assertThat(response, hasProperty("mobileApplication.ios.minimalVersion", equalTo("1.5.4")));
-        assertThat(response, hasProperty("mobileApplication.iso.currentVersion", equalTo("2.0.0")));
+        assertThat(response.getMobileApplication().getAndroid().getMinimalVersion(), equalTo("1.4.0"));
+        assertThat(response.getMobileApplication().getAndroid().getCurrentVersion(), equalTo("1.5.4"));
+        assertThat(response.getMobileApplication().getIOs().getMinimalVersion(), equalTo("1.5.4"));
```

Comment:
We use the `getIos()` convention, let's stick to it for consistency, see e.g.:
https://github.com/wultra/powerauth-push-server/blob/develop/powerauth-push-model/src/main/java/io/getlime/push/model/entity/PushServerApplication.java#L50

Reply:
Ok, but I have to change it in the upstream.

Reply:
We can do it later... not a real issue.

Reply:
Merging, to have at least some test. Will be improved later on.